protocol/testutil/constants/subaccounts.go

```diff
@@ -80,8 +108,9 @@ var (
 		},
 		PerpetualPositions: []*satypes.PerpetualPosition{
 			{
-				PerpetualId: 0,
-				Quantums:    dtypes.NewInt(-10_000_000_000), // -100 BTC
+				PerpetualId:  0,
+				Quantums:     dtypes.NewInt(-10_000_000_000), // -100 BTC
+				FundingIndex: dtypes.NewInt(0),
```

Comment:
Why were all of these changed? Doesn't seem related to this PR?

Reply:
I have it in the description:
>cleaned up some tech debt I found where perpetual positions should be initialized with a 0 funding index rather than the default value for the byte array for test constants

Mainly did this because I had to change it for the constants I used, which made the constants file inconsistent.